### PR TITLE
[Liquidstakeibc] C Value Fix

### PR DESCRIPTION
**Pull request by kruspy**

### CODE REVIEW

File: x/liquidstakeibc/keeper/deposit.go [hunk 1]
```diff
@@ -234,3 +234,43 @@ func (k *Keeper) GetDelegatingDepositsForChain(ctx sdk.Context, chainID string)
...
+	iterator := sdk.KVStorePrefixIterator(store, nil)
+	defer iterator.Close()
+
+	amount := sdk.ZeroInt()
+	for ; iterator.Valid(); iterator.Next() {
+		deposit := &liquidstakeibctypes.Deposit{}
+		k.cdc.MustUnmarshal(iterator.Value(), deposit)
+
+		if deposit.ChainId == chainID &&
+			(deposit.State == liquidstakeibctypes.Deposit_DEPOSIT_RECEIVED ||
```
Comment (puneet2019) [inline x/liquidstakeibc/keeper/deposit.go:269]: I think even SENT you want to add here?

Comment (kruspy) [reply x/liquidstakeibc/keeper/deposit.go:269]: Yeah that one is kind on the middle, anyway wherever we put it it is going to work

File: x/liquidstakeibc/keeper/keeper.go [hunk 2]
```diff
@@ -262,3 +262,73 @@ func (k *Keeper) SendICATransfer(
...
+				hc.CValue,
+			),
+		)
+
+		hc.LastCValue = hc.CValue
+		hc.CValue = cValue
+		k.SetHostChain(ctx, hc)
+
+		// if the c value is out of bounds, disable the chain
+		if hc.CValue.GT(k.GetParams(ctx).UpperCValueLimit) || hc.CValue.LT(k.GetParams(ctx).LowerCValueLimit) {
```
Comment (puneet2019) [inline x/liquidstakeibc/keeper/keeper.go:316]: why not use CheckCValueWithinLimits function?

Comment (kruspy) [reply x/liquidstakeibc/keeper/keeper.go:316]: Lol I missed that

**Review by puneet2019 — APPROVED** [review]

**Pull request merged**